wrangler-core/src/main/java/io/cdap/directives/aggregates/IncrementTransientVariable.java

```diff
@@ -71,7 +71,7 @@ public void initialize(Arguments args) throws DirectiveParseException {
     try {
       el.compile(expression);
     } catch (ELException e) {
-      throw new DirectiveParseException(e.getMessage());
+      throw new DirectiveParseException("Error encountered while initializing '" + NAME + "' : " + e.getMessage());
```

Comment:
When wrapping exception, always includes the origin exception cause. Otherwise debugging would be extremely difficult.

Reply:
Agreed. This behavior is same as existing code. Fixing it for the directives.